vendor/k8s.io/kubernetes/pkg/kubectl/cmd/cmd.go

```diff
@@ -304,6 +304,10 @@ type PluginHandler interface {
 
 type defaultPluginHandler struct{}
 
+func NewDefaultPluginHandler() *defaultPluginHandler {
```

Comment:
Use this constructor in `NewDefaultKubectlCommand` too.

Reply:
Other option is to use a global var similar to `ValidPluginFilenamePrefixes` and use it in both needed places.

Reply:
Yeah, I'm looking into that and just re-using that same constant here will be the simplest approach and will be also easier to justify upstream :wink: